content/articles/oop-in-java-abstraction-and-encapsulation/index.md

```diff
@@ -0,0 +1,456 @@
+ 
+### The Concept of OOP in Java: Getting Started with Abstraction and Encapsulation
+ 
+Object-Oriented Programming (OOP) is a programming paradigm (approach), as is procedural programming. Unlike procedural programming, we model everything as objects (real-life entities). Java, as an object-oriented programming language, was designed to follow this pattern.
+ 
+Objects come from a class, a form of generalization, but a real thing is an object. What exactly is a class? A class is a template to create objects.
+The process of creating objects from a class is called instantiation.
+ 
+In this tutorial, you will learn about abstraction, encapsulation, objects, and classes. The concept of OOP will be a nice piece to grab for absolute Java beginners.
+
+### Prerequisites
+The reader should have:
+- At least a basic knowledge of Java
+- Have Java installed on your computer
+- Have any IDE of your choice
+ 
+### Objectives
+At the end of this first OOP tutorial series, readers should be able to:
+- Differentiate between classes and objects
+- Understand abstraction and encapsulation
+- Create objects from classes
+- Have a working knowledge of what OOP is about
+- Create abstract classes and interfaces
+ 
+### Class
+ 
+A class is a template or a blueprint for object creation.
+To represent attributes or properties, classes use instance variables whereas, for behaviors or actions, it uses instance methods.
+ 
+#### Instance Variables
+These are variables declared in the class, outside of methods and constructors. Instance variables have a global scope.
+ 
+#### Instance Methods
+These are methods that only an object of its class can invoke.
+It is the class that defines what an object data field or variables and methods will be.
+An example of a class is a circle class.
+A circle has a radius from which we can create different objects of type circle.
+The following code snipet explains this idea.
```

Comment:
There is a spelling error here. Change *snipet* to *snippet*.

Reply:
Done